variables.tf

```
@@ -49,3 +49,8 @@ variable "routes" {
   description = "List of routes being created in this VPC"
   default     = []
 }
+
+variable "delete_default_internet_gateway_routes" {
+  description = "If set, ensure that all routes within the network specified whose names begin with 'default-route' and with a next hop of 'default-internet-gateway' are deleted"
+  default     = ""
```

This should be treated as a boolean.

```suggestion
  default     = "false"
```